### PR TITLE
Mandrel July 2023 update

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Hum, the CI failure is related, the signature is wrong:
```
[INFO] java.lang.IllegalStateException: Invalid signature for artifact mandrel-java20-linux-aarch64-23.0.1.2-Final.tar.gz, e6ed1f76292a5f44bff076225ebbb503b940180b14d6468d8722769ff7eb9e9e != be992959df83beb9f5fe0ffe4606d76b2deba6cb86322b9d3614f09a8a2fbbc8
```

**Comment by zakkak** [comment]
Hmmm, that's interesting:

```
graalvm-jdk-17.0.8-linux-x64.tar.gz, 905899607dc8bea924921a33b413bc5b01bc7c804501bffca3e06f9a7a432838 != 1dffdf5c7cc5bf38558e9f093eef6a14072a6dff0be3a9906208b37b53ecc009
```

`1dffdf5c7cc5bf38558e9f093eef6a14072a6dff0be3a9906208b37b53ecc009` is indeed the sha256 [published on github](https://github.com/graalvm/graalvm-ce-builds/releases/download/jdk-17.0.8/graalvm-community-jdk-17.0.8_linux-x64_bin.tar.gz.sha256)

**Comment by cescoffier** [comment]
@zakkak hum... yeah... something is off...

File: quarkus-mandrel-builder-image/mandrel.yaml [hunk 1]
```diff
@@ -34,3 +35,21 @@ images:
         arch: amd64
       - sha: 3179c87dc9336a0283a5da2c3ba74ddbd4fbec52d9d8dd78cb6979b34ce8464a
         arch: arm64
+
+  - graalvm-version: 23.0.1.2-Final
+    java-version: 17
+    tags: 23.0-java17, 23.0-jdk-17, jdk-17, jdk-17.0.7
```
Comment (Karm) [inline quarkus-mandrel-builder-image/mandrel.yaml:41]: We need fixed tags, 17.0.8

File: quarkus-mandrel-builder-image/mandrel.yaml [hunk 2]
```diff
@@ -34,3 +35,21 @@ images:
...
+    variants:
+      - sha: cc73adfae094d05367e008d8ee3eebf7a08a3b45b504e55771dc739a7faabe3b
+        arch: amd64
+      - sha: 373d8341cfa678e3bdffeb8936c3edd8c353b19fdd54ba4cbe6711c6c1faa127
+        arch: arm64
+
+  - graalvm-version: 23.0.1.2-Final
+    java-version: 20
+    tags: 23.0-java20, 23.0-jdk-20, jdk-20, jdk-20.0.1
```
Comment (Karm) [inline quarkus-mandrel-builder-image/mandrel.yaml:50]: This tag is also in error, 20.0.2

File: quarkus-mandrel-builder-image/mandrel.yaml (hunk 1 above)
Comment (zakkak) [reply quarkus-mandrel-builder-image/mandrel.yaml:41]: Handled in https://github.com/quarkusio/quarkus-images/pull/247

File: quarkus-mandrel-builder-image/mandrel.yaml (hunk 2 above)
Comment (zakkak) [reply quarkus-mandrel-builder-image/mandrel.yaml:50]: Handled in https://github.com/quarkusio/quarkus-images/pull/247

**Pull request merged**